spec/requests/purchases/tipping_spec.rb

```diff
@@ -16,28 +16,31 @@
       add_to_cart(product2)
       fill_checkout_form(product2)
 
-      expect(page).to have_radio_button("0%", checked: true)
+      expect(page).to have_radio_button("No Tip", checked: true)
+      expect(page).to have_text("Add a tip? US$0", normalize_ws: true)
 
-      choose "10%"
+      choose "20%"
 
+      expect(page).to have_field("Custom tip", with: "")
       expect(page).to have_text("Subtotal US$40", normalize_ws: true)
-      expect(page).to have_text("Tip US$4", normalize_ws: true)
-      expect(page).to have_text("Total US$44", normalize_ws: true)
+      expect(page).to have_text("Tip US$8", normalize_ws: true)
+      expect(page).to have_text("Total US$48", normalize_ws: true)
+      expect(page).to have_text("Add a tip? US$8", normalize_ws: true)
```

Comment:
[NEW] To check tip value is rendered besides the seciton header updates when tip is changed